substrate/client/informant/src/lib.rs

```diff
@@ -51,41 +51,15 @@ impl Default for OutputFormat {
 	}
 }
 
-enum ColorOrStyle {
-	Color(Colour),
-	Style(Style),
-}
-
-impl From<Colour> for ColorOrStyle {
-	fn from(value: Colour) -> Self {
-		Self::Color(value)
-	}
-}
-
-impl From<Style> for ColorOrStyle {
-	fn from(value: Style) -> Self {
-		Self::Style(value)
-	}
-}
-
-impl ColorOrStyle {
-	fn paint(&self, data: String) -> impl Display {
-		match self {
-			Self::Color(c) => c.paint(data),
-			Self::Style(s) => s.paint(data),
-		}
-	}
-}
-
 impl OutputFormat {
 	/// Print with color if `self.enable_color == true`.
-	fn print_with_color(
-		&self,
-		color: impl Into<ColorOrStyle>,
-		data: impl ToString,
-	) -> impl Display {
+	fn print(&self, color: Color, attr: Option<Attribute>, data: impl Display) -> impl Display {
```

Comment:
@bkchr `console` API design is not good here. Do you have any suggestions?

Reply:
What is the problem here? I don't see it.

Reply:
In `ansi_term`, the function can just pass `Colour::Red.bold()` to get red and bold style, but `console` doesn't provide such fluent API, I separate `color` and `attribute`, and because there are usages do not have `attribute`, so I have to make it optional which feels not good than previous one

Reply:
I think this is fine. This is some internal api any way.